app/controllers/order_controller.py

```diff
@@ -31,8 +30,8 @@ def process_order(cls, data):
         return_object = {"message": "Commande traitée avec succès"}
 
         product = data.get('product', {})
-        id = product.get('id', {})
-        quantity = product.get('quantity', {})
+        id = int(product.get('id', {}))
+        quantity = int(product.get('quantity', {}))
```

Comment:
This seem wrong, we should not give a {} then convert it to a int if nothing is there.
```suggestion
        id = product.get('id', 0)
        quantity = product.get('quantity', 0)
```
Something like the above look better, however im not sure if 0 is the best way since it could maybe be an id.